Tasks/AzureAppServiceManageV0/azureappservicemanage.ts

```diff
@@ -202,4 +216,84 @@ async function run() {
     }
 }
 
+async function handleActionWithPublishProfileEndpoint(publishProfileEndpoint: AzureEndpoint, action: string, extensionList: string, extensionOutputVariables: string) {
+    let scmCreds: scmCredentials = await getSCMCredentialsFromPublishProfile(publishProfileEndpoint.PublishProfile);
+    let kuduService = new Kudu(scmCreds.uri, scmCreds.username, scmCreds.password);
+    let kuduServiceUtils = new KuduServiceUtils(kuduService);
+    let taskResult = true;
+    let errorMessage: string = "";
+
+    try {
+        switch(action) {
+            case "Start all continuous webjobs": {
+                await kuduServiceUtils.startContinuousWebJobs();
```

Comment:
we should use the existing code to handle various actions. kuduService and kuduServiceutils should be different based on auth scheme. looks possible to do so if kudu service is created appropriately in `azureAppServiceUtils.getKuduService()`